cpp/src/arrow/util/int_util_internal.h

```diff
@@ -63,9 +63,9 @@ OPS_WITH_OVERFLOW(DivideWithOverflow, div)
 #undef OP_WITH_OVERFLOW
 #undef OPS_WITH_OVERFLOW
 
-// Define functions NegateWithOverflow with the signature `bool(T u, T* out)`
-// where T is a signed integer type.  On overflow, these functions return true.
-// Otherwise, false is returned and `out` is updated with the result of the
+// Define functions NegateWithOverflow with the signature
+// `bool(T u, T* out)` where T is a signed integer type.  On overflow, these functions
+// return true. Otherwise, false is returned and `out` is updated with the result of the
```

Comment:
Nit: is this intended or spurious change?

Reply:
It was a spurious change because I had initially added an `AbsoluteValueWithOverflow` but `NegateWithOverflow` sufficed.